test/e2e/performanceprofile/functests/utils/deployments/deployments.go

```diff
@@ -0,0 +1,91 @@
+package deployments
+
+import (
+	"context"
+	"time"
+
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/images"
```

Comment:
Please rearrange imports in the following order:

standarads libs

k8s libs

third-part libs

To be more specific  Openshift imports should go under the k8s imports